src/wallet/gtest/test_orchard_wallet.cpp

```diff
@@ -128,7 +132,7 @@ TEST(TransactionBuilder, OrchardToOrchard) {
     auto orchardAnchor = tree.root();
 
     // Create an Orchard-only transaction
-    // 0.0004 z-ZEC in, 0.00025 z-ZEC out, default fee, 0.00005 z-ZEC change
+    // 0.0004 z-ZEC in, 0.00025 z-ZEC out, 0.0001 fee, 0.00005 z-ZEC change
```

Comment:
Why is this 0.0001 and not 0.00001?

Reply:
You’re right. I “corrected” this in the wrong direction. Instead of modifying the fee, I should have modified the change amount.

```suggestion
    // 0.0004 z-ZEC in, 0.00025 z-ZEC out, default fee, 0.00014 z-ZEC change
```